image/mkosi.skeleton/usr/lib/udev/google_nvme_id

```diff
@@ -0,0 +1,247 @@
+
+
```

Comment:
Are these newlines at the top part of the script, or were they added in by accident?
They also appear in the other gce scripts

Reply:
Thanks. :)